angular/README.md

```diff
@@ -1,6 +1,6 @@
-# Getting Started with DevExtreme Angular NumberBox
+# DevExtreme Angular NumberBox - Getting Started 
 
-This repository stores the source code for the following tutorial: [Getting Started with NumberBox](https://js.devexpress.com/Documentation/Guide/UI_Components/NumberBox/Getting_Started_with_NumberBox/).
+This repository stores the code examples of the Angular NumberBox component for the [Getting Started with NumberBox](https://js.devexpress.com/Documentation/Guide/UI_Components/NumberBox/Getting_Started_with_NumberBox/) tutorial. The component allows users to enter a value and increment or decrement it with the spin buttons, keyboard or mouse wheel.
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.0.2.
 
```

Comment:
Please update the following line below:

Lines 21 and 25:
Please change "via" to "using".